src/subdomains/core/sell-crypto/buy-fiat/buy-fiat.service.ts

```diff
@@ -97,6 +99,19 @@ export class BuyFiatService {
       await this.userService.activateUser(entity.sell?.user);
     }
 
+    // payment webhook
+    // TODO add fiatFiatUpdate here
+    if (dto.inputAmount && dto.inputAsset) {
+      entity.sell
+        ? await this.webhookService.cryptoFiatUpdate(entity.sell.user.userData, entity, PaymentWebhookState.CREATED)
+        : null;
+    }
+    if (dto.outputAmount && dto.outputAsset) {
+      entity.sell
+        ? await this.webhookService.cryptoFiatUpdate(entity.sell.user.userData, entity, PaymentWebhookState.COMPLETED)
+        : null;
+    }
```

Comment:
Hm. We will send a both webhooks, if outputAmount and outputAsset is set, won't we? Maybe something like
```ts
if (dto.outputAmount && dto.outputAsset && entity.sell) {
   await this.webhookService.cryptoFiatUpdate(entity.sell.user.userData, entity, PaymentWebhookState.COMPLETED);
    } else if (dto.inputAmount && dto.inputAsset && entity.sell) {
   await this.webhookService.cryptoFiatUpdate(entity.sell.user.userData, entity, PaymentWebhookState.CREATED);
}
```